docs/devel/README.md

```diff
@@ -401,10 +401,13 @@ mkdir /usr/share/containers/storage: read-only file system
 
```

Comment:
I assume this should move -> docs/devel/experimental/SUBPACKAGES.md
Isnt it? 

I assume you the generic one is under ./README.md customize QM setting

Reply:
Yeah, that makes sense to me. Done